src/lazy/binary/raw/v1_1/type_descriptor.rs

```diff
@@ -9,6 +9,7 @@ pub struct Opcode {
     pub opcode_type: OpcodeType,
     pub ion_type: Option<IonType>,
     pub low_nibble: u8,
+    pub byte: u8,
```

Comment:
🗺️ When reading an e-expression in binary, the opcode byte is often the ID itself. Now the `Opcode` captures the original byte so we can inspect it later.

I briefly looked at removing the `low_nibble` field, thinking I could get that value by masking the `byte` field. I realized in the process of putting this PR together that the `low_nibble` field is not _actually_ the low nibble of the opcode (though it often is). It represents either the value length or, if a long encoding is being used, will be `0xFF`.

We'll probably want to change that field name back and document its intent.